tests/src/Common/test_runtime/project.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "Microsoft.NETCore.Platforms": "2.0.0-beta-25001-02",
```

Comment:
We already have that dependency there - the issue (as @dagood pointed out) is that one of the dependencies in this file (and TestWrappersConfig's project.json) is bringing in an older version of Microsoft.NETCore.Platforms, that didn't support Ubuntu16.10/ Ubuntu16.04/ Fedora23.